src/libraries/listener.library.ts

```diff
@@ -155,7 +163,29 @@ export class KeyboardShortcutListener implements IKeyboardShortcutListener {
 
         // check vs blacklist
         if (blackListedKeyboardShortcutChecker) {
-            blackListedKeyboardShortcutChecker.check(this);
+            let areWeChecking: boolean = null;
+            // check listener
+            if (this.checkBlackList === true) {
```

Comment:
You can avoid this if / else by just assigning the areWeChecking to the this.checkBlackList.  Something like this

```typescript
            let areWeChecking = this.checkBlackList;

            // if not set check config
            if (areWeChecking === null) {
                areWeChecking = config.defaultCheckBlackList;
            }
            // if true run check
            if (areWeChecking === null || areWeChecking) {
                blackListedKeyboardShortcutChecker.check(this);
            }
```